src/test/spec/command_monitoring/operation.rs

```diff
@@ -110,42 +110,6 @@ impl TestOperation for DeleteOne {
     }
 }
 
-// This struct is necessary because the command monitoring tests specify the options in a very old
-// way (SPEC-1519).
-#[derive(Debug, Deserialize, Default)]
-struct FindModifiers {
-    #[serde(rename = "$comment", default)]
-    comment: Option<String>,
-    #[serde(rename = "$hint", default)]
-    hint: Option<Hint>,
-    #[serde(
-        rename = "$maxTimeMS",
-        deserialize_with = "bson_util::deserialize_duration_option_from_u64_millis",
-        default
-    )]
-    max_time: Option<Duration>,
-    #[serde(rename = "$min", default)]
-    min: Option<Document>,
-    #[serde(rename = "$max", default)]
-    max: Option<Document>,
-    #[serde(rename = "$returnKey", default)]
-    return_key: Option<bool>,
-    #[serde(rename = "$showDiskLoc", default)]
-    show_disk_loc: Option<bool>,
-}
-
-impl FindModifiers {
-    fn update_options(&self, options: &mut FindOptions) {
-        options.comment = self.comment.clone();
-        options.hint = self.hint.clone();
-        options.max_time = self.max_time;
-        options.min = self.min.clone();
-        options.max = self.max.clone();
-        options.return_key = self.return_key;
-        options.show_record_id = self.show_disk_loc;
-    }
-}
-
 #[derive(Debug, Default, Deserialize)]
 pub(super) struct Find {
```

Comment:
Can we add `#[serde(deny_unknown_fields)]` to this to ensure we don't miss any new fields?

Reply:
Done.